README.md

```diff
@@ -156,7 +154,7 @@ You can find more detail in our [Contributing guide](CONTRIBUTING.md). Participa
 ## 📃 License
 
 `bpmn-visualization` is released under the [Apache 2.0](LICENSE) license. \
-Copyright &copy; 2020, Bonitasoft S.A.
+Copyright &copy; 2020-2021, Bonitasoft S.A.
```

Comment:
maybe it is better to put something like:
2020 - now
or set up some reminder in calendar to keep it up to date

Reply:
we have decided to use 'from 2020'